modules/cli/cmd/sequencer/set-config.go

```diff
@@ -0,0 +1,118 @@
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+	Run:   setAssetCmdHandler,
+}
+
+func setAssetCmdHandler(c *cobra.Command, args []string) {
+	flagHandler := cmd.CreateCliFlagHandler(c, cmd.EnvPrefix)
+
+	sequencerNetworksConfigPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", DefaultSequencerNetworksConfigFilename)
+	// create the networks config file if it doesn't exist. Will skip if the
+	// file already exists.
+	CreateSequencerNetworkConfigs(sequencerNetworksConfigPath)
+	sequencerNetworkConfigs := LoadSequencerNetworkConfigsOrPanic(sequencerNetworksConfigPath)
+
+	network := flagHandler.GetValue("network")
+	config.IsInstanceNameValidOrPanic(network)
+
+	asset := args[0]
+	config.IsInstanceNameValidOrPanic(asset)
+
+	log.Info("Updating sequencer_chain_id to", asset, "in sequencer-networks-config.toml for network: ", network)
+	config := sequencerNetworkConfigs.Configs[network]
+	config.Asset = asset
+	config.FeeAsset = asset
+	sequencerNetworkConfigs.Configs[network] = config
+
+	file, err := os.Create(sequencerNetworksConfigPath)
+	if err != nil {
+		log.Error("Error creating file:", sequencerNetworksConfigPath, ":", err)
+		panic(err)
+	}
+	defer file.Close()
+
+	// Replace the networks config toml with the new config
+	if err := toml.NewEncoder(file).Encode(sequencerNetworkConfigs); err != nil {
+		panic(err)
+	}
+	log.Infof("Successfully updated networks-config.toml: %s\n", sequencerNetworksConfigPath)
+}
+
+// setSequencerChainIdCmd represents the set-config sequencer-chain-id command
+var setSequencerChainIdCmd = &cobra.Command{
+	Use:   "sequencer-chain-id [chain-id]",
+	Short: "Sets the sequencer chain id in the sequencer command configs.",
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+	Run:   setSequencerChainIdCmdHandler,
+}
+
+func setSequencerChainIdCmdHandler(c *cobra.Command, args []string) {
+	flagHandler := cmd.CreateCliFlagHandler(c, cmd.EnvPrefix)
+
+	sequencerNetworksConfigPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", DefaultSequencerNetworksConfigFilename)
+	// create the networks config file if it doesn't exist. Will skip if the
+	// file already exists.
+	CreateSequencerNetworkConfigs(sequencerNetworksConfigPath)
+	sequencerNetworkConfigs := LoadSequencerNetworkConfigsOrPanic(sequencerNetworksConfigPath)
+
+	network := flagHandler.GetValue("network")
+	config.IsInstanceNameValidOrPanic(network)
+
+	sequencerChainId := args[0]
+	config.IsInstanceNameValidOrPanic(sequencerChainId)
+
+	log.Info("Updating sequencer_chain_id to", sequencerChainId, "in sequencer-networks-config.toml for network: ", network)
+	config := sequencerNetworkConfigs.Configs[network]
+	config.SequencerChainId = sequencerChainId
+	sequencerNetworkConfigs.Configs[network] = config
+
+	file, err := os.Create(sequencerNetworksConfigPath)
+	if err != nil {
+		log.Error("Error creating file:", sequencerNetworksConfigPath, ":", err)
```

Comment:
You can combine this line and the next line I think with something like `log.Fatal` or log.Panic`